system_test/ResetTest.words.log

```diff
@@ -0,0 +1,500 @@
+> > frame #: 1
```

Comment:
This log doesn't look right: it should be showing where the object(s) are and what they're saying.